headers/types/dungeon_mode/dungeon_mode.h

```diff
@@ -990,9 +993,10 @@ struct dungeon_generation_info {
     bool dough_seed_extra_poke_flag;
     // 0x5: Room index of Monster House on the floor. 0xFF if there's no Monster House
     uint8_t monster_house_room;
-    // 0x6: Related to when a monster from a fixed room faints. Maybe to check if the floor
-    // should be over after knocking them out?
-    undefined unk_fixed_room_static_monster_tracker;
+    // 0x6: It's set to 0 when it's not a fixed room, 1 when it's a fixed room with no boss fight,
+    // and 2 when it's a fixed room with a boss fight. If this tracker is 0, 1, or 3 the floor
```

Comment:
So is 3 also a legitimate value? The first sentence makes it seem like 0, 1, and 2 are the only allowed values.

Reply:
The game specifically checks for 3 when passing the floors, but I couldn't find anywhere the game sets it to 3. Could be related to a debug thing or I just missed another location which does set it to 3.

Reply:
I see, I guess this is fine.